source/guide_searx.rst

```diff
@@ -0,0 +1,242 @@
+
+Some python modules are necessary and will be installed in your uberspace with ``pip3`` and the option ``--user``:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ pip3 install --user requests pyyaml pygments werkzeug babel flask flask_babel lxml langdetect python-dateutil
+
+Step 3 - Searx Configuration
+----------------------------
+
+We create the configuration file destination directory:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ mkdir ~/etc/searx
+
+And copy the example file with basic default settings to the new directory:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cp ~/opt/searx/utils/templates/etc/searx/use_default_settings.yml ~/etc/searx/settings.yml
+
+Now it's time to change some entries in the configuration file:
+
+1. Searx requires for the own instance a secret key. This random number will be created with openssl (16 digits) and placed direct in the config file via sed, the stream editor.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ sed -i -e "s/ultrasecretkey/`openssl rand -hex 16`/g" ~/etc/searx/settings.yml
+
+
+2. You can change the name of your own searx instance. The standard name is "searx".
+
+3. The bind address must be changed to ``0.0.0.0``, to work with `web backends`_ in a common way.
+
+----
+
+.. note:: The port with ``8888`` will not be touched, but keep this number in your mind for later configurtations.
+
+.. code-block:: yaml
+ :emphasize-lines: 5,13,14,15
+
+use_default_settings: True
+
+general:
+    debug : False # Debug mode, only for development
+    instance_name : "searx" # displayed name
+
+search:
+    safe_search : 0 # Filter results. 0: None, 1: Moderate, 2: Strict
+    autocomplete : "" # Existing autocomplete backends: "dbpedia", "duckduckgo", "google", "startpage", "swisscows", "qwant", "wikipedia" - leave blank to turn it off by default
+    default_lang : "" # Default search language - leave blank to detect from browser information or use codes from 'languages.py'
+
+server:
+    port : 8888
+    bind_address : "0.0.0.0" # address to listen on
+    secret_key : "01234567890123456" # change this!
+    base_url : False # Set custom base_url. Possible values: False or "https://your.custom.host/location/"
+    image_proxy : False # Proxying image results through searx
+
+# uncomment below section if you have running morty proxy
+#result_proxy:
+#    url : http://127.0.0.1:3000/
+#    key : !!binary "your_morty_proxy_key"
+
+
+Step 4 - Supervisord Setup
+--------------------------
+
+At first we must create the service file ``~/etc/services.d/searx.ini`` and the following content:
+
+.. warning:: The PATH variable consider the installed Python modules regarding to the version 3.6 at this time point. Please have a look in the directory ``~/.local/lib/`` to be sure which version is the actual ones.
+
+.. code-block::
+ :emphasize-lines: 2
+
+[program:searx]
+environment =
+ PATH="%(ENV_HOME)s/opt/searx/:%(ENV_HOME)s/.local/lib/python3.6/:%(ENV_PATH)s",
```

Comment:
`PATH` specifies where to find binaries to run, but  `~/.local/lib/python3.6` and `~/opt/searx` both don't contain any binaries. Are you sure this is needed?

Reply:
I had in a Python project a problem with environment settings to find the external modules (libraries). But in this case everthing is fine without the PATH.